server/session.py

```diff
@@ -587,3 +594,95 @@ def notify(self, height, touched):
             self.mns.add(vin)
             return result.get(vin)
         return None
+
+class PacElectrumX(DashElectrumX):
```

Comment:
Is there any reason this new code does not apply to Dash too?

Reply:
there's no reason, that code can apply to Dash as well, I just didn't want to modify it (what if I broke something), but shall I add it to the DashElectrumX class? 

btw the code was tested with the PAC network and works well

Reply:
Yes please add it to DashElectrumX